### PR TITLE
small fix of utxo calculation on raw transaction build

### PR DESCRIPTION
**Pull request by aprasolova**
When one builds raw transaction, one needs to calculate outputs from assets transfer. If sender's UTXO is spent totally, then there is no need to include zero unspent into the transaction outputs. Now, in transaction compose this case isn't considered.

### CODE REVIEW

File: pkg/rpc/txBuilder.go [hunk 1]
```diff
@@ -48,8 +48,11 @@ func CreateRawContractTransaction(params ContractTxParams) (*transaction.Transac
 		tx.AddInput(&input)
 	}
 
-	senderOutput = transaction.NewOutput(assetID, spent-amount, fromAddressHash)
-	tx.AddOutput(senderOutput)
+	senderUnspent := spent - amount
+	if senderUnspent > 0 {
+		senderOutput = transaction.NewOutput(assetID, senderUnspent, fromAddressHash)
+		tx.AddOutput(senderOutput)
+	}
```
Comment (im-kulikov) [inline pkg/rpc/txBuilder.go]: `if senderUnspent := spent - amount; senderUnspent > 0 {`

Comment (aprasolova) [reply pkg/rpc/txBuilder.go]: done

**Comment by hal0x2328** [comment]
Award aprasolova 100 points
Award im-kulikov 20 points for review

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/144dc06fbc54dd84165af08ddd9c01e8a720ca1bb05471dfd1af1503160f245f

**Pull request merged**